README.md

```diff
@@ -1,83 +1,36 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
+# Takeaway Challenge
 
-Instructions
--------
+## About
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+An IRB application to order a takeaway. Created as the weekend challenge for week 2 of the Makers course.
 
-Task
------
+Uses the Twilio API to send a confirmation text. You will need to sign up for your own trial account to use this.
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+Although this app is used through irb, it was built with the idea that it could be implemented with its own UI. Because of this some features are unintuitive. E.g. To add a dish to the basket you must pass the dish object. You can access these through the menu class.
 
-```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
+## How to run
 
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
+First set up a Twilio account here: https://www.twilio.com/.
 
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
```

Comment:
The user currently doesn't get to verify that their order is correct. I think that would be a great feature to add.